web/registry/models.py

```diff
@@ -145,6 +137,12 @@ def create_from_dict(component_spec_dict: Dict) -> List:
             # TODO - When we have accounts, we need to check the the user
             #        has permission to create a new version of this Component
             #        (i.e. if the name already exists but not the version).
+            print(Component.objects.all())
```

Comment:
This print statement probably won't be great as the DB grows. :)

Reply:
Removed